frontend/src/pages/admin/FlowBlueprints/dialogs/FlowBlueprintFormDialog.vue

```diff
@@ -179,17 +179,33 @@ export default {
 
             // Validation
             try {
-                flowBlueprintProps.flows = JSON.parse(flowBlueprintProps.flows)
+                JSON.parse(flowBlueprintProps.flows)
             } catch (err) {
                 this.error = 'Invalid JSON for flows'
+                this.errors.flows = 'Invalid JSON'
+                return
+            }
+
+            try {
+                if (!JSON.parse(flowBlueprintProps.flows).flows) {
+                    throw new Error('Flow json missing \'flows\' property')
+                }
+                if (!Array.isArray(JSON.parse(flowBlueprintProps.flows).flows)) {
+                    throw new Error('Flow json \'flows\' property not an Array')
+                }
+            } catch (err) {
+                this.error = err
                 this.errors.flows = err
                 return
             }
+
+            flowBlueprintProps.flows = JSON.parse(flowBlueprintProps.flows)
+
             try {
                 flowBlueprintProps.modules = JSON.parse(flowBlueprintProps.modules)
             } catch (err) {
                 this.error = 'Invalid JSON for modules'
-                this.errors.modules = err
+                this.errors.modules = 'Invalid JSON'
```

Comment:
Suggestion - tell the users what we expect:

```suggestion
                this.errors.modules = 'Modules should be an object of "module" : "version" pairs'
```

or something less technical (but equally informative)?